TrackingTools/Producers/src/StraightLinePropagatorESProducer.cc

```diff
@@ -33,7 +33,7 @@ StraightLinePropagatorESProducer::produce(const TrackingComponentsRecord & iReco
 
   PropagationDirection dir = alongMomentum;
 
-  if (pdir == "oppositeToMomentum, alongMomentum, anyDirection")
+//  if (pdir == "oppositeToMomentum, alongMomentum, anyDirection")
     if (pdir == "oppositeToMomentum") dir = oppositeToMomentum;
     if (pdir == "alongMomentum") dir = alongMomentum;
     if (pdir == "anyDirection") dir = anyDirection;
```

Comment:
While you're at it, why not add `else`:
```
if (pdir == "oppositeToMomentum") dir = oppositeToMomentum;
else if (pdir == "anyDirection") dir = anyDirection;
 ```
`if` for `alongMomentum` not needed since it's already the default value.